rules_factory.rb

```diff
@@ -0,0 +1,19 @@
+class RulesFactory
+
+  def initialize
+  end
+
+  def find_rule_for(item)
+    if ("Aged Brie".eql?(item.name))
```

Comment:
instead of the long if/else, I've much prefer to see a case statement instead. item.name duplication as well as the elsif's go away.